pkg/resolver/search.go

```diff
@@ -160,7 +194,7 @@ func (s *SearchResult) resolveItems() ([]map[string]interface{}, error) {
 	if err != nil {
 		klog.Errorf("Error resolving query [%s] with args [%+v]. Error: [%+v]", s.query, s.params, err)
 	}
-	defer rows.Close()
+	// defer rows.Close()
```

Comment:
We need to call row.Close() to avoid leaking connections in the pool.

Reply:
forgot to uncomment, thanks.